api/v01/vrp.rb

```diff
@@ -163,18 +163,19 @@ def self.vrp_request_vehicle(this)
-        this.optional(:coef_service, type: Float, desc: 'Coefficient applied to every service duration defined in the tour, for this vehicle')
-        this.optional(:additional_service, type: Float, desc: 'Constant additional service time for all travel defined in the tour, for this vehicle')
+        this.optional(:cost_time_multiplier, type: Float, desc: 'Cost applied to the total amount of time of travel (Jsprit) or to the total time of route (ORtools). Not taken into account within periodic heuristic.')
+        this.optional(:cost_value_multiplier, type: Float, desc: 'Multiplier applied to the value matrix and additional activity value. Not taken into account within periodic heuristic.')
+        this.optional(:cost_waiting_time_multiplier, type: Float, desc: 'Cost applied to the waiting time in the route. Not taken into account within periodic heuristic.')
+        this.optional(:cost_late_multiplier, type: Float, desc: 'Cost applied if a point is delivered late (ORtools only). Not taken into account within periodic heuristic.')
+        this.optional(:cost_setup_time_multiplier, type: Float, desc: 'Cost applied on the setup duration (Jsprit only). Not taken into account within periodic heuristic.')
+        this.optional(:coef_setup, type: Float, desc: 'Coefficient applied to every setup duration defined in the tour, for this vehicle. Not taken into account within periodic heuristic.')
+        this.optional(:additional_setup, type: Float, desc: 'Constant additional setup duration for all setup defined in the tour, for this vehicle. Not taken into account within periodic heuristic.')
+        this.optional(:coef_service, type: Float, desc: 'Coefficient applied to every service duration defined in the tour, for this vehicle. Not taken into account within periodic heuristic.')
+        this.optional(:additional_service, type: Float, desc: 'Constant additional service time for all travel defined in the tour, for this vehicle. Not taken into account within periodic heuristic.')
         this.optional(:force_start, type: Boolean, documentation: { hidden: true }, desc: '[ DEPRECATED ]')
-        this.optional(:shift_preference, type: String, values: ['force_start', 'force_end', 'minimize_span'], desc: 'Force the vehicle to start as soon as the vehicle timewindow is open, as late as possible or let vehicule start at any time. Not available with periodic heuristic.')
-        this.optional(:trips, type: Integer, desc: 'Describe the number of return to the depot a vehicle is allowed to perform within its route')
+        this.optional(:shift_preference, type: String, values: ['force_start', 'force_end', 'minimize_span'], desc: 'Force the vehicle to start as soon as the vehicle timewindow is open,
+          as late as possible or let vehicule start at any time. Not available with periodic heuristic, it will always leave as soon as possible.')
```

Comment:
```suggestion
          as late as possible or let vehicle start at any time. Not available with periodic heuristic, it will always leave as soon as possible.')
```

Reply:
Is this option still active? I thought we disactivated `force_start` (which looks hidden). @braktar  do you know what's the deal with this param?